parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -168,6 +168,10 @@ configureTransparentHugePage
 configureSwapFile
 {{- end}}
 
+{{- if ShouldConfigureHTTPProxyCA}}
+configureHTTPProxyCA
+{{- end}}
+
```

Comment:
this should comes as the first step before any service runs, otherwise they will fail (such as apt-get install), right? So we should move it to around L42

Reply:
Makes sense. moved to beginning